src/App/Fossa/LicenseScanner.hs

```diff
@@ -351,11 +352,11 @@ licenseScanSourceUnit ::
   ) =>
   VendoredDependencyScanMode ->
   Maybe LicenseScanPathFilters ->
-  FullFileUploads ->
+  FileUpload ->
   Path Abs Dir ->
   NonEmpty VendoredDependency ->
   m (NonEmpty Locator)
-licenseScanSourceUnit vendoredDependencyScanMode licenseScanPathFilters fullFileUploads baseDir vendoredDeps = do
+licenseScanSourceUnit mode licenseScanPathFilters upload baseDir vendoredDeps = do
```

Comment:
I think renaming this to `skipMode` would make future-me have to think less about what `mode` means

Reply:
Done!